<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DslamPageWidget</class>
 <widget class="QWidget" name="DslamPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topButtonPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="backToBoardListButton">
        <property name="text">
         <string>Назад</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/prev.png</normaloff>:/images/prev.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="boardNameLabel">
        <property name="text">
         <string>Доска X</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>460</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="finishEditButton">
        <property name="text">
         <string>Окончить редактирование</string>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="beginEditButton">
        <property name="toolTip">
         <string>Перейте в режим редактирования</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="editToolButtonPanel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="editBoardFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QCheckBox" name="autoUpdateBoardListCheckBox">
           <property name="text">
            <string>Автоматическое получение списка досок</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoNumeringPairCheckBox">
           <property name="text">
            <string>Автоматическая нумерация пар</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QToolButton" name="editBoardButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="removeBoardButton">
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/remove.png</normaloff>:/images/remove.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="operationToolButtonPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="upPortButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/up.png</normaloff>:/images/up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="downPortButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resource.qrc">
          <normaloff>:/images/down.png</normaloff>:/images/down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="dslamTreeView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="profileFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="profileFrameLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Профиль</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="closeProfileFrameButton">
          <property name="toolTip">
           <string>Закрыть</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resource.qrc">
            <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="profileListComboBox"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="applyProfileButton">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="upPortAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/up.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Активировать порт</string>
   </property>
   <property name="toolTip">
    <string>Активировать порт</string>
   </property>
  </action>
  <action name="downPortAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/down.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Деактивировать порт</string>
   </property>
   <property name="toolTip">
    <string>Деактивировать порт</string>
   </property>
  </action>
  <action name="refreshPortInfoAction">
   <property name="text">
    <string>Обновить информацию по порту</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию по порту</string>
   </property>
  </action>
  <action name="refreshAllPortInfoAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/refresh.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Обновить информацию по всем портам</string>
   </property>
   <property name="toolTip">
    <string>Обновить информацию по всем портам</string>
   </property>
  </action>
  <action name="collapseAllNodeAction">
   <property name="text">
    <string>Свернуть все узлы</string>
   </property>
   <property name="toolTip">
    <string>Свернуть все узлы</string>
   </property>
  </action>
  <action name="showBoardAction">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть</string>
   </property>
  </action>
  <action name="showProfileFrameAction">
   <property name="text">
    <string>Изменить профиль</string>
   </property>
   <property name="toolTip">
    <string>Изменить профиль</string>
   </property>
  </action>
  <action name="editBoardAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/edit.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Изменить текущую ячейку</string>
   </property>
   <property name="toolTip">
    <string>Изменить текущую ячейку</string>
   </property>
  </action>
  <action name="removeBoardAction">
   <property name="icon">
    <iconset>
     <normalon>:/images/remove.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Удалить текущую доску</string>
   </property>
   <property name="toolTip">
    <string>Удалить текущую доску</string>
   </property>
  </action>
  <action name="renumeringPairAction">
   <property name="text">
    <string>Перенумеровать пары</string>
   </property>
   <property name="toolTip">
    <string>Перенумеровать пары</string>
   </property>
  </action>
  <action name="getBoardListAction">
   <property name="text">
    <string>Запрос досок с дслама</string>
   </property>
   <property name="toolTip">
    <string>Запрос досок с дслама</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
